pkg/trace/obfuscate/json.go

```diff
@@ -58,9 +79,21 @@ func (p *jsonObfuscator) setKey() {
 	p.wiped = false
 }
 
+func trimSideQuotes(quotedString []byte) []byte {
```

Comment:
Would `strconv.Unquote` work here?

Reply:
Oh didn't know about that. It works. Updated.